.env_sample

```diff
@@ -117,7 +117,7 @@ HONEYCOMB_API_KEY=""
 
 # Google analytics
 # (https://developers.google.com/analytics/devguides/reporting/core/v4)
-GA_TRACKING_ID="Optional"
+# GA_TRACKING_ID="Optional"
```

Comment:
elsewhere we have either an empty string, or a blank variable, would this make as much sense? 

```
GA_TRACKING_ID=
```

Unless the variable being _defined_ is a problem, it's coloring a little outside the lines to comment the variable out

Reply:
I think the behavior is that if you set ENV then the Settings module uses that, otherwise, whatever's in the DB?  (That's what I've been able to sleuth.)